### DISCUSSION TITLE
Links add title

### DISCUSSION THREAD

**Discussion by viktorgusev1984** [likec4/likec4#866] [Ideas]
Now links are an opportunity to navigate, but it is not clear where the link leads in a meaningful way.
It will be very convenient if we add a title that will be displayed instead of the URL

now it displays like this
<img width="457" alt=" example1" src="https://github.com/user-attachments/assets/d09ae58b-bc0b-46bd-8997-b0310d858f0f">

what do I suggest

```
model {
  service = component {
    link http://jira.com/S1 'Create by Task S1'
    link http://confluence.com/wertyui 'Docs'
  }
}
```
<img width="451" alt="example2" src="https://github.com/user-attachments/assets/cfe9af7f-5397-4b98-81f4-b024020094b9">

**Comment by davydkov** [comment]
@viktorgusev1984 Nice idea

Created #871
